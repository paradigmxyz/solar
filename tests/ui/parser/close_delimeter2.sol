struct X {
    //~v ERROR: expected one of `;` or `}`, found `<eof>`
    uint y