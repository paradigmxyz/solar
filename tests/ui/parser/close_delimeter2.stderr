error: expected one of `;` or `}`, found `<eof>`
  --> ROOT/tests/ui/parser/close_delimeter2.sol:LL:CC
   |
LL |     uint y
   |          ^ expected one of `;` or `}`

error: aborting due to 1 previous error

